packages/roosterjs-editor-plugins/lib/plugins/Paste/sanitizeHtmlTextFromPastedContent/sanitizeHtmlTextFromPastedContent.ts

```diff
@@ -0,0 +1,27 @@
+import { chainSanitizerCallback, getTagOfNode } from 'roosterjs-editor-dom';
+import { DeprecatedColorList } from './deprecatedColorList';
+import { HtmlSanitizerOptions } from 'roosterjs-editor-types';
+
+/**
+ * @internal
+ * Remove the deprecated colors from pasted content
+ * @sanitizingOption
+ * */
+export default function sanitizeHtmlTextFromPastedContent(
+    fragment: DocumentFragment,
+    sanitizingOption: Required<HtmlSanitizerOptions>
+) {
+    const htmlElements = fragment.querySelectorAll('*') as NodeListOf<HTMLElement>;
```

Comment:
Not sure if I am correct, but if there are many repeated Elements with tags, we would add many repeated chainSanitizerCallbacks.

I think we can filter the unique tags doing:

```
const allTags = fragment.querySelectorAll('*').map(el => getTagOfNode(el));
const uniqueTags = allTags.filter((tag, index)=>  allTags.indexOf(tag) == index) )
```

And then use unique tags to add the chainSanitizeCallbacks as you are doing below.